src/Post_Command.php

```diff
@@ -173,6 +180,21 @@ public function create( $args, $assoc_args ) {
 		$array_arguments = array( 'meta_input' );
 		$assoc_args      = \WP_CLI\Utils\parse_shell_arrays( $assoc_args, $array_arguments );
 
+            if( isset( $assoc_args['from-post'] ) ) {
+                  $post     = $this->fetcher->get_check( $assoc_args['from-post'] );
+                  $post_arr = get_object_vars( $post );
+                  $post_id  = $post_arr['ID'];
+                  unset( $post_arr['post_date'] );
+                  unset( $post_arr['post_date_gmt'] );
+                  unset( $post_arr['guid'] );
```

Comment:
WHy did you decide to skip these values?

Reply:
This values will be duplicated too. I skip post_date and post_date_gmt so that values can be reset according to the current date at the time of duplicating. Reason to unset guid is that duplicated post guid will same as the original.

If you have any suggestions please let me know.